backend/src/main/java/com/google/models/Timestamp.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Wrapper for LocalDate object with day, month, year fields. 
+ * Accepts format in yyyy-MM-dd or yyyy-M-d or yyyy-MM-d or yyyy-M-dd, but prints format in yyyy-MM-dd.
+ */
+public final class Timestamp {
```

Comment:
If this `Timestamp` class is only used for `Date`, do we want to modify the class name?

or did you want to keep it as an extensible `Timestamp` which might have more granular (non-daily) breakdown in the future?

Reply:
I feel like if we keep the name as `Timestamp` it will be more extensible in the future in case the data we get have time units more granular than daily.

Reply:
ok sgtm